src/eventStore/dynamodb/index.ts

```diff
@@ -0,0 +1,132 @@
+import * as dynamoose from 'dynamoose';
+import { AbiItem, Contract, EventStoreConfig, IEventStore } from '../interface';
+import { AntennaModel } from './antenna';
+
+export class EventStoreDynamoDB implements IEventStore {
+  config: EventStoreConfig['dynamodb'];
+
+  constructor(config: EventStoreConfig['dynamodb']) {
+    this.config = config;
+  }
+
+  async connect(): Promise<void> {
+    const config: any = {
```

Comment:
If it is possible, please describe minimal type for using dynamoDB.
`{ region: string; secretAccessKey?: string; accessKeyId?: string; }` will be enough for now.

Reply:
At this point, the parameter type of the EventStoreClass constructor is never.